package.json

```diff
@@ -28,10 +29,12 @@
     "@babel/plugin-proposal-class-properties": "^7.3.4",
     "chokidar": "^3.0.0",
     "cookie-parser": "^1.4.4",
+    "esm": "^3.2.25",
```

Comment:
Adding ESM here so we can use es6 modules natively without any weirdness. Using it in the start script above - `node -r esm`